test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -199,18 +199,18 @@ function RunUpdateHelpTests
         if ($powershellCoreModules -contains $moduleName)
         {
 
-            It "Validate Update-Help for module '$moduleName' with scope as '$userscope'" -Skip:(!(Test-CanWriteToPsHome) -and $userscope -eq $false) {
+            if($userscope)
+            {
+                $params = @{Path = $testCases[$moduleName].HelpInstallationPathHome}
+                $updateScope = @{Scope = 'CurrentUser'}
+            }
+            else
+            {
+                $params = @{Path = $testCases[$moduleName].HelpInstallationPath}
+                $updateScope = @{Scope = 'AllUsers'}
+            }
 
-                if($userscope)
-                {
-                    $params = @{Path = $testCases[$moduleName].HelpInstallationPathHome}
-                    $updateScope = @{Scope = 'CurrentUser'}
-                }
-                else
-                {
-                    $params = @{Path = $testCases[$moduleName].HelpInstallationPath}
-                    $updateScope = @{Scope = 'AllUsers'}
-                }
+            It "Validate Update-Help for module '$moduleName' in '$updateScope'" -Skip:(!(Test-CanWriteToPsHome) -and $userscope -eq $false) {
```

Comment:
'$updateScope' returns "System.Collections.Hashtable". Yes?

Reply:
I am deeply ashamed of myself.  I assumed it would be nicely expanded in a string.  I am working on it.

Reply:
We could simply add new variable for the test name.

Reply:
I would rather be as deep to the metal as possible in a test log.